bucketlist_app/static/script.js

```diff
@@ -0,0 +1,33 @@
+// Get the modal
```

Comment:
Is this called with the onload or onready function?

Reply:
I'm not making use of jquery at all for it. It's just JS.